doc/tutorials/dnn/dnn_openvino/dnn_openvino.markdown

```diff
@@ -0,0 +1,29 @@
+OpenCV usage with OpenVINO {#tutorial_dnn_openvino}
+=====================
+
+@prev_tutorial{tutorial_dnn_halide_scheduling}
+@next_tutorial{tutorial_dnn_android}
+
+|    |    |
+| -: | :- |
+| Original author | Aleksandr Voron |
+| Compatibility | OpenCV == 4.x |
+
+# Quick start {#tutorial_openvino_install_introduction}
+
+Since 2021.1.1 release OpenVINO does not provide pre-built OpenCV. 
+The change does not affect you if you are using OpenVINO runtime directly or OpenVINO samples: it does not have a strong dependency to OpenCV.
+However, if you are using Open Model Zoo demos or OpenVINO runtime as OpenCV DNN backend you need to get the OpenCV build.
+
+There are 2 approaches how to get OpenCV:
+
+- Install prebuilt OpenCV from another sources: system repositories, pip, conda, homebrew. This approach has several limitations:
+    - OpenCV version is out-of-date
+    - OpenCV does not contain G-API module (e.g. some OMZ demos use G-API functionality)
+    - OpenCV does not use available CPU instructions since it has build to cover wide range of hardware
+    - OpenCV does not support Intel TBB, Intel Media SDK
```

Comment:
> OpenCV does not

Generic OpenCV package may not

Reply:
Instead of adding it to all bullets I've changed the 20th line: `Generic pre-built OpenCV package may have several limitations` I believe it addresses your comment